src/components/drag-and-drop/draggable-context/draggable-context.js

```diff
@@ -146,7 +148,12 @@ class DraggableContext extends React.Component {
    * Renders the component
    */
   render() {
-    return this.props.children;
+    return (
+      <div className='carbon-draggable-context'>
+        { this.props.children }
+        <CustomDragLayer />
```

Comment:
Could this pass some DragLayerProps so that its easier to change the custom drag layer.

Or enable a way so that you can optionally pass your own custom drag layer.